content/millennium/r4/base/individuals/related-person.md

```diff
@@ -193,6 +193,33 @@ _Implementation Notes_
 
 #### Response
 
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Fri, 16 Jul 2021 17:23:14 GMT
+Etag: W/"0"
+Location: https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson/204989530-12724068
```

Comment:
Can we update the location ID to  12724066-12724068 as the POST body resembles that relatedperson?

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/689/commits/451cbf433fbbf86d7ca24f7a0509b1885e471318#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584R203

Reply:
Thanks!